test/mocks/mocks.go

```diff
@@ -118,3 +131,56 @@ func SecretMock(ns, name string) corev1.Secret {
 		},
 	}
 }
+
+// ServiceBindingRequestMock return a binding-request mock of informed name and match labels.
+func ServiceBindingRequestMock(
+	ns, name, resourceRef string, matchLabels map[string]string,
+) v1alpha1.ServiceBindingRequest {
+	return v1alpha1.ServiceBindingRequest{
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
Should we add TypeMeta here?

```
               TypeMeta: metav1.TypeMeta{
                       Kind:       OperatorKind,
                       APIVersion: OperatorAPIVersion,
               },
```

Reply:
Based on [this](https://github.com/operator-framework/operator-sdk/blob/e0342c4f1c34e961520956baaadb5851cc11a8ac/doc/user/unit-testing.md#testing-with-3rd-party-resources) I haven't add the `TypeMeta` over there. I'm not sure is needed, to be honest.